doc/engineering-reference/src/climate-sky-and-solar-shading-calculations/climate-calculations.tex

```diff
@@ -60,17 +60,44 @@ \subsection{Sky Radiation Modeling}\label{sky-radiation-modeling}
 
 Temperature\(_{drybulb}\) = drybulb temperature \{K\}
 
-The sky emissivity is given by
+By default, the clear sky emissivity is calculated by the Clark \& Allen (1978) model as
 
 \begin{equation}
-Sk{y_{emissivity}} = \left( {.787 + .764\cdot \ln \left( {\frac{{Temperatur{e_{dewpoint}}}}{{273.}}} \right)} \right)\cdot \left( {1. + .0224N - .0035{N^2} + .00028{N^3}} \right)
+Sky_{clear-sky-emissivity} = 0.787 + 0.764\cdot \ln \frac {T_{dewpoint}}{T_{Kelvin}}
 \end{equation}
 
-where
+where, T\(_{dewpoint}\) = dewpoint temperature \{K\}.
+
+Alternatively, the clear sky emissivity can also be calculated using different models set by the user from options using the \textbf{WeatherProperty:SkyTemperature} object, including the calibrated version of the Berdahl and Martin, Brunt and Idso models (2017), listed as follows.
+
+Calibrated Berdahl and Martin model:
```

Comment:
Minor, but I've usually seen this referenced as Martin Berdahl rather than Berdahl Martin. Please check recent literature to see what's the most common citation of this model. It may depend on whether one is referencing this 1983 LBNL report https://escholarship.org/content/qt71v36429/qt71v36429.pdf or the 1984 paper you've referenced. Are the equations the same in both references?

Reply:
They use the same equation for sky emissivity.  

Emissivity of clear skies - Berdahl and Martin (https://doi.org/10.1016/0038-092X(84)90144-0) (cited 142) is the first to introduced the emissivity equation.  Martin and Berdahl (https://doi.org/10.1016/0038-092X(84)90162-2) (cited 158) introduced the sky radiant temperature equation from this emissivity estimation.

I still tend to call it Berdahl and Martin because *ESP-r* that currently uses this algorithm calls it Berdahl and Martin algorithm. (http://www.esru.strath.ac.uk/Programs/ESP-r_FAQ.htm)

Reply:
Ok